### DISCUSSION TITLE
Subtract Coulomb interaction

### DISCUSSION THREAD

**Discussion by caperott** [ttadano/alamode#235] [Q&A] [answered]
Hi @ttadano, 
I am interested in fitting an IFC-based model to a (partially) ionic compound. Is there any way (using Born charges) the Coulomb contribution to the forces between atoms be subtracted before fitting the IFCs (and then added back for calculating the phonon spectrum)?

**Comment by ttadano** [comment, marked as answer]
Hi,

A similar treatment is implemented in the anphon code and automatically used when `NONANALYTIC=3`. 

In this method, we first subtract the dipole-dipole contributions from the second-order force constants in the supercell and then add the dipole-diple term back to the dynamical matrix in the Fourier space.

In anphon, setting `FC2_EWALD = 1` in the &analysis field along with proper `BORNFILE` and `NONANALYTIC=3` will make PREFIX.fc2_ewald, which contains dipole-dipole contributions and the remaining part of harmonic force constants.

While it is possible to do a similar decomposition for atomic forces, but we have not implemented it.

**Reply by caperott** [reply]
Hi @ttadano,
Thank you very much for your prompt reply!
Best regards,
Claudio
